src/routes/docs/products/auth/+layout.svelte

```diff
@@ -97,6 +97,10 @@
                     label: 'Multi-factor authentication',
                     href: '/docs/products/auth/mfa'
                 },
+                {
+                    label: 'Checking auth status',
```

Comment:
Let's keep the tense consistent with all the others.

Reply:
@stnguyen90 Would "Authentication status" work?

Reply:
Or "Auth status check"?